charts/argo-cd/templates/argocd-server/alb-grpc-service.yaml

```
@@ -0,0 +1,26 @@
+{{- if .Values.server.ingressGrpc.enabled -}}
+{{- if .Values.server.ingressGrpc.isAWSALB -}}
```

Is there a reason, why you didn't write it like this?
```suggestion
{{- if and .Values.server.ingressGrpc.enabled .Values.server.ingressGrpc.isAWSALB -}}
```

---

Nope, no reason, I can change it to the one line if statement.

---

Changed the if statements.